src/mame/drivers/pacman.cpp

```diff
@@ -1595,11 +1595,11 @@ static INPUT_PORTS_START( mspacman )
 	PORT_BIT( 0x02, IP_ACTIVE_LOW, IPT_JOYSTICK_LEFT ) PORT_4WAY
 	PORT_BIT( 0x04, IP_ACTIVE_LOW, IPT_JOYSTICK_RIGHT ) PORT_4WAY
 	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_JOYSTICK_DOWN ) PORT_4WAY
-	PORT_DIPNAME( 0x10, 0x10, "Rack Test (Cheat)" ) PORT_CODE(KEYCODE_F1)
+	PORT_DIPNAME( 0x10, 0x10, "Rack Test (Cheat)" ) PORT_CODE(KEYCODE_F1) PORT_DIPLOCATION("SW:7")
 	PORT_DIPSETTING(    0x10, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_COIN1 )
-	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_COIN2 )
+	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_COIN1 ) /* Also invincibility in mspacpls */
+	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_COIN2 ) /* Also speed-up in mspacpls */
```

Comment:
Coin inputs give invincibility and speed-up? Was this a copy/paste meant to be on INP_START1/2?

Also may as well make these C++ style comments while we're moving them :wink:

Reply:
makes me wonder if these were multiplexed somehow, or if they're really just debug features.  A moment of speedup and/or invincibility when inserting a coin seems like an odd choice.

Reply:
Ah yeah, was a silly copy/paste mistake. Fixed and converted the comments